test/rage2e/rag_test.go

```diff
@@ -471,114 +502,178 @@ func createAndValidateIndexPod(ragengineObj *kaitov1alpha1.RAGEngine) error {
 				return false
 			}
 
-			logs, err := utils.GetPodLogs(coreClient, ragengineObj.Namespace, "indexpod", "")
+			logs, err := utils.GetPodLogs(coreClient, ragengineObj.Namespace, podName, "")
 			if err != nil {
-				GinkgoWriter.Printf("Failed to get logs from pod %s: %v\n", "indexpod", err)
+				GinkgoWriter.Printf("Failed to get logs from pod %s: %v\n", podName, err)
 				return false
 			}
 
 			return strings.Contains(logs, "Kaito is an operator that automates the AI/ML model inference or tuning workload in a Kubernetes cluster")
-		}, 2*time.Minute, utils.PollInterval).Should(BeTrue(), "Failed to wait for index logs to be ready")
+		}, 10*time.Minute, utils.PollInterval).Should(BeTrue(), "Failed to wait for index logs to be ready")
```

Comment:
Is it necessary to change it to 10 minutes? I think 10 minutes is too long

Reply:
Yes this is the max timeout, it will proceed sooner if it succeeds. I increased it once because curl pod was stuck in container creating past two minutes. But I  agree we should lower it, lowered to 4mins.